app/search_builders/curation_concerns/filter_suppressed_with_roles.rb

```diff
@@ -0,0 +1,27 @@
+module CurationConcerns
+  module FilterSuppressedWithRoles
+    extend CurationConcerns::FilterSuppressed
+
+    def only_active_works(solr_parameters)
+      if current_work.respond_to?(:to_sipity_entity) # Used because Collections use WorkSearchBuilder to filter down to a single item
+        super if user_work_permissions.empty?
```

Comment:
So would the change here be to `include` FilterSuppressed rather than extend it? Or is there another pattern or technique I should be using here?